examples/use_cases/creating_reusable_components/reusing_previous_work.ipynb

```diff
@@ -0,0 +1,902 @@
+    "id": "pJJpAMl5QFMu"
+   },
+   "source": [
+    "We will now train different models on this dataset. \n",
+    "\n",
+    "Scaling all values to be within the (0, 1) interval will reduce the distortion due to exceptionally high values and make some algorithms converge faster."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {
+    "_cell_guid": "23fc8398-331d-4164-8779-0516264ece29",
+    "_uuid": "c147b9e899cc2dd508d736274c856f88fb49321d",
+    "id": "33WC-SSjQFMu"
+   },
+   "outputs": [],
+   "source": [
+    "scaler = MinMaxScaler()\n",
+    "sdss = scaler.fit_transform(sdss_df_fe.drop('class', axis=1))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "_cell_guid": "13e1fd6f-820c-4098-a486-0079b300e6c9",
+    "_uuid": "e76e964a4b93c27e1ab01c24a06be09d8b304970",
+    "id": "MZlx5sj6QFMu"
+   },
+   "source": [
+    "We will  split the data into a training and a test part. The models will be trained on the training data set and tested on the test data set"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {
+    "_cell_guid": "a94f2305-4f2a-4d11-b368-528820d28b0b",
+    "_uuid": "da9a96996b695f8ca2c268f43106e179ba6be1f5",
+    "id": "Xz0znPYjQFMu"
+   },
+   "outputs": [],
+   "source": [
+    "X_train, X_test, y_train, y_test = train_test_split(sdss, sdss_df_fe['class'], test_size=0.33)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "models = {\n",
+    "    \"K Nearest Neighbors Classifier\": KNeighborsClassifier(),\n",
+    "    \"Gaussian Naive Bayes Classifier\": GaussianNB(),\n",
+    "    \"Random Forest Classifier\": RandomForestClassifier(n_estimators=10),\n",
+    "    \"Random Forest Classifier x100\": RandomForestClassifier(n_estimators=100),\n",
+    "    \"Support Vector Machine Classifier\": SVC(),\n",
+    "}"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# save these variables as artifacts because they will be used again and again for different experiments\n",
+    "lineapy.save(X_train, \"X_train\")\n",
+    "lineapy.save(X_test, \"X_test\")\n",
+    "lineapy.save(y_train, \"y_train\")\n",
+    "lineapy.save(y_test, \"y_test\")\n",
+    "lineapy.save(models, \"models\");"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Consider a typical model training below using one of the models in our dictionary"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "\n",
+    "modelname = \"K Nearest Neighbors Classifier\"\n",
+    "model = models[modelname]\n",
+    "\n",
+    "model.fit(X_train, y_train)\n",
+    "preds = model.predict(X_test)\n",
+    "# model accuracies\n",
+    "acc = (preds == y_test).sum().astype(float) / len(preds)*100\n",
```

Comment:
sklearn.metrics.accuracy_score?

Reply:
leaving this as is from the original notebook (since we are comparing for the demo)